roomba_980/__init__.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+# vim: set encoding=utf-8 tabstop=4 softtabstop=4 shiftwidth=4 expandtab
+#########################################################################
+#  Copyright 2011 KNX-User-Forum e.V.           http://knx-user-forum.de/
+#########################################################################
+#  This file is part of SmartHome.py.    http://mknx.github.io/smarthome/
+#
+#  SmartHome.py is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  SmartHome.py is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with SmartHome.py. If not, see <http://www.gnu.org/licenses/>.
+#########################################################################
+
+import logging
+from plugins.roomba_980.roomba import Roomba
+from lib.model.smartplugin import SmartPlugin
+from lib.item import Items
+
+class ROOMBA_980(SmartPlugin):
+
+    ALLOW_MULTIINSTANCE = False
+    PLUGIN_VERSION = "1.0.0"
+
+    logger = logging.getLogger(__name__)
```

Comment:
Logging is [already initialized](https://github.com/smarthomeNG/smarthome/blob/develop/lib/model/smartplugin.py#L59) by `SmartPlugin`.